README.md

```diff
@@ -173,3 +173,7 @@ There are a few additional changes needed to `.env` when moving to production.
 Review the Laravel documentation on deploying Horizon to a production environment.
 
 Also be sure to set up a cron job to run scheduled tasks - Horizon uses this to keep track of statistics.
+
+# Security reporting
+
+Any security issues with the code or any RoboJackets-managed deployment (*.robojackets.org) should be reported to [apiary@robojackets.org](mailto:apiary@robojackets.org). This will notify our development and operations teams and you should receive a response within 8 business hours.
```

Comment:
You're asking for security issues with any RJ website to go to an address specific for Apiary